lang/ghc/Portfile

```diff
@@ -1,16 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           compiler_blacklist_versions 1.0
 
 name                ghc
-# Do not update GHC separate from Haskell Platform.
-# When updating GHC, make sure to revbump all Haskell ports.
-# Also make sure to update the version in the Haskell PortGroup.
```

Comment:
These comments were deleted but it doesn't seem like required actions were documented anywhere else or were rendered unnecessary: I am currently unable to build something with the new ghc because all existing `hs-*` ports' binaries install for ghc 7.8.3.

What is the intended solution?